src/main/java/com/perficient/movies/model/dtos/ActorDTO.java

```diff
@@ -0,0 +1,23 @@
+package com.perficient.movies.model.dtos;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
+@Setter
+@Getter
+public class ActorDTO {
+
+    private String name;
+
+    private String lastName;
+
+    private String bio;
+
+    private boolean isActive;
+
+    private int startYear;
+
+    private String biggestRole;
+}
```

Comment:
You are not using this dto in your controller

Reply:
Got it. 👍